src/test/data/entity/fullAdmission.ts

```diff
@@ -0,0 +1,257 @@
+  repaymentPlan: {
+    completionDate: MomentFactory.currentDate().subtract(12, 'months'),
+    firstPaymentDate: MomentFactory.currentDate().subtract(1, 'days'),
+    instalmentAmount: 10,
+    paymentLength: '10 months',
+    paymentSchedule: PaymentSchedule.EVERY_MONTH
+  }
+}
+
+export const installmentsPaymentIntention = {
+  paymentOption: PaymentOption.INSTALMENTS,
+  repaymentPlan: {
+    completionDate: MomentFactory.currentDate().add(12, 'months'),
+    firstPaymentDate: MomentFactory.currentDate().add(15, 'days'),
+    instalmentAmount: 10,
+    paymentLength: '10 months',
+    paymentSchedule: PaymentSchedule.EVERY_MONTH
+  }
+
+}
+
+export const courtDeterminationBySpecifiedDate = {
+  courtDecision: {
+    ...setByDatePaymentIntention
+  },
+  courtPaymentIntention: {
+    paymentDate: MomentFactory.maxDate(),
+    paymentOption: PaymentOption.BY_SPECIFIED_DATE,
+    decisionType: 'DEFENDANT',
+    rejectionReason: 'test'
+  }
+}
+
+export const courtDeterminationInInstalments = {
+  courtDecision: {
+    ...installmentsPaymentIntention
+  },
+  courtPaymentIntention: {
+    paymentDate: MomentFactory.maxDate(),
+    paymentOption: PaymentOption.BY_SPECIFIED_DATE,
+    decisionType: 'DEFENDANT',
+    rejectionReason: 'test'
+  }
+}
+
+export const claimantReferredToJudgeResponse = {
+  claimantPaymentIntention: { ...immediatePaymentIntention },
+  courtDetermination: { ...courtDeterminationBySpecifiedDate },
+  formaliseOption: FormaliseOption.REFER_TO_JUDGE,
+  type: ClaimantResponseType.ACCEPTATION
+}
+
+export const claimantReferredToJudgeResponseForInstalments = {
+  claimantPaymentIntention: { ...immediatePaymentIntention },
+  courtDetermination: { ...courtDeterminationInInstalments },
+  formaliseOption: FormaliseOption.REFER_TO_JUDGE,
+  type: ClaimantResponseType.ACCEPTATION
+}
+
+export const claimantAcceptRepaymentPlan = {
+  formaliseOption: FormaliseOption.SETTLEMENT,
+  type: ClaimantResponseType.ACCEPTATION
+}
+
+export const claimantAcceptRepaymentPlanByDetermination = {
+  claimantPaymentIntention: { ...immediatePaymentIntention },
+  courtDetermination: { ...courtDeterminationBySpecifiedDate },
+  ...claimantAcceptRepaymentPlan
+}
+
+export const claimantAcceptRepaymentPlanInInstalmentsByDetermination = {
+  claimantPaymentIntention: { ...immediatePaymentIntention },
+  courtDetermination: { ...courtDeterminationInInstalments },
+  ...claimantAcceptRepaymentPlan
+}
+
+export const claimantResponseAt = {
+  claimantRespondedAt: MomentFactory.currentDate()
+}
+
+export const defendantOffersSettlementBySetDate = [{
+  type: 'OFFER',
+  madeBy: 'DEFENDANT',
+  offer: {
+    content: 'test',
+    completionDate: MomentFactory.currentDate().add(1, 'day'),
+    paymentIntention: { ...setByDatePaymentIntention }
+  }
+}]
+
+export const defendantOffersSettlementByInstalments = [{
+  type: 'OFFER',
+  madeBy: 'DEFENDANT',
+  offer: {
+    content: 'test',
+    completionDate: MomentFactory.currentDate().add(1, 'day'),
```

Comment:
this date should probably be further in the future if it is instalments.. and isn't there also a completion date inside the paymentIntention? shouldn't they be the same?

Reply:
Yes it should be the same I have updated to future after 12 months